test/paragraph-cases/linebreaks.html

```diff
@@ -0,0 +1,2 @@
+<p>if a paragraph contains a linebreak, it should preserve the linebreak in the output (before beautification) and even if a line starts with a
```

Comment:
This is not the output I would have anticipated. I would have thought either a space between "in" and "the" and also between "a" and "format", or a \n between the two. What we see here is a space between the former and a \n between the latter. Seems weird.